lib/mayaUsd/utils/util.h

```diff
@@ -580,6 +601,22 @@ float ConvertMTimeUnitToFloat(const MTime::Unit& unit);
 MAYAUSD_CORE_PUBLIC
 float GetSceneMTimeUnitAsFloat();
 
+/**
+ * Searches the given array for an element.
+ *
+ * @param a         The element to search for.
+ * @param array     The array to search within.
+ * @param idx       Storage for the index of the element within the array if it exists.
+ *                  If it does not exist, this will be undefined.
```

Comment:
Maybe mention that `idx` can be `nullptr` if you're only interested in whether the array contains the element and don't care what the index is, and/or make it optional with `unsigned int* idx = nullptr`?